src/guide/working-with-the-legacy-mercurial-servers.md

```diff
@@ -221,7 +221,7 @@ In order to push changesets into the parent repository, some additional configur
 
 ### Get your SSH key installed
```

Comment:
Should this header use title capitalisation as other headers, at least in this review?

Reply:
No, currently only first level headers are capitalized in this document.